.gitignore

```diff
@@ -114,3 +114,75 @@ env/
 
 ### Configuration files ###
 .env
+
```

Comment:
FYI we're expecting people to use VSCode for the project exercise long term (so at the very least it's worth checking your project still runs from VSCode before each submission)